tests/Browser/SignoutTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
```

Comment:
Rename deze test naar signoutTrainingTest ofzo want straks hebben we een login pagina waar je wss ook moet uitloggen, daar is deze naam beter voor.

Reply:
fixed